### ISSUE TITLE
Monster Mode - Random Walk

### ISSUE THREAD

**Issue by Emistry** [HerculesWS/Hercules#1527]
I think this mode is missing or not implemented.
Or does it changed to other?

I think there is a mode that doesn't allow monster to random walk around when idle.
or doesn't random walk around when there are no players around them.

**Comment by AnisotropicDefixation** [comment]
Yeah, that's present in some new instance. rA has that implemented so it's easy to merge.

**Comment by Emistry** [comment]
> I believe `setunitdata()` solves this; can anyone confirm?

yes, tried this method recently, it does solve the issue. 
Set the monster mode with `CanWalk + NoRandomWalk`.

**Comment by Emistry** [comment]
my bad, it was actually a different branch... currently Herc don't have these field.
